src/services/health-check/presenter-result-builder.php

```diff
@@ -0,0 +1,207 @@
+<?php
+
+namespace Yoast\WP\SEO\Services\Health_Check;
+
+/**
+ * Provides an interface to build WordPress-friendly health check results.
+ */
+class Presenter_Result_Builder {
```

Comment:
```suggestion
class Result_Presenter { ??
class Result_Builder { ??
```
een builder presenter doet twee dingen; dat is niet solid. laten we een betere naam verzinnen.

Reply:
Report Builder